es_mapping.json

```diff
@@ -1,45 +1,46 @@
-{"properties": {
-				"url": {
-					"type": "text"
-				},
-				"zoneName": {
+{
+	"properties": {
```

Comment:
I think this file is provided to help admins setup the mappings when first creating the index.

Nothing references this file, so that must be its purpose.